cmd/cve/generate.go

```diff
@@ -44,11 +45,22 @@ func main() {
 		panic(err)
 	}
 
+	// Download the last released NVD feed
+	logger.Log("msg", "Downloading latest release")
+	if err := downloadLatestRelease(*dbDir, *debug, logger); err != nil {
+		panic(err)
+	}
+
 	// Sync the CVE files
 	if err := nvd.GenerateCVEFeeds(*dbDir, *debug, logger); err != nil {
 		panic(err)
 	}
 
+	// Remove Vulncheck archive
+	if err := os.RemoveAll(filepath.Join(*dbDir, "vulncheck.zip")); err != nil {
+		logger.Log("msg", "Failed to remove vulncheck.zip", "err", err)
+	}
```

Comment:
Won't removing it here make rm fail [here](https://github.com/fleetdm/vulnerabilities/blob/83a628f616f9a14bb16c7756198609416bd12c5c/.github/workflows/generate-cve.yml#L61)?

(Maybe add a `rm -rf` on the other repo)

Reply:
good catch, i fixed that in my personal repo.  i'll have a PR up in a few.

Reply:
https://github.com/fleetdm/vulnerabilities/pull/8